libs/designer/src/lib/ui/panel/panelTabs/parametersTab/index.tsx

```diff
@@ -235,12 +235,18 @@ const ParameterSection = ({
     },
     getDisplayValueFromSelectedItem: (selectedItem: any): string => {
       const dependency = dependencies.inputs[parameter.parameterKey];
-      const propertyPath = dependency.filePickerInfo?.fullTitlePath ?? dependency.filePickerInfo?.browse.itemFullTitlePath;
+      const propertyPath =
+        dependency.filePickerInfo?.fullTitlePath ??
+        dependency.filePickerInfo?.browse.itemFullTitlePath ??
+        dependency.definition?.extension?.dynamicState?.dynamicExtension?.browse.itemFullTitlePath;
```

Comment:
Is there a better way to pass or get the paths from dynamicState?

Reply:
There should be one place where paths are atleast for one extension why do we have two fallbacks for dynamictree?

Reply:
These callbacks are provided so UI code does not assume how to get these values.

Reply:
Removed changes from this file.